cuda/matrix/csr_kernels.cu

```diff
@@ -303,59 +303,58 @@ void spmv(std::shared_ptr<const CudaExecutor> exec,
             syn::value_list<int>(), syn::type_list<>(), exec, a, b, c);
     } else if (a->get_strategy()->get_name() == "sparselib" ||
                a->get_strategy()->get_name() == "cusparse") {
-        if (cusparse::is_supported<ValueType, IndexType>::value) {
```

Comment:
Does cusparse support int64 in those routines we use?
If it is, another way is to update cusparse::is_supported according to the cuda version.

Reply:
I guess the difference here is that cuSPARSE itself will throw an exception of a combination is not supported with the Generic API, while in older versions, you had differently named functions for different types, and no 64 bit support at all. As long as we have both types of routines in Ginkgo, we need to differentiate between them.